website/src/test/java/com/google/step/OrganizationInfoTest.java

```diff
@@ -0,0 +1,35 @@
+package com.google.step;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.data.OrganizationInfo;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class OrganizationInfoTest {
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
```

Comment:
Consider adding a test for `merge`. And also `isValid`.

Sometimes you want to just get up and running with a prototype and hold off on rich tests until things become more clear (e.g. what fields actually are required, how do we want to handle merging for duplicated entries).

In general, it's much cheaper to just add the tests now and update them with the changing implementations than to have to frantically debug something untested going wrong down the line.